### ISSUE TITLE
Choose EPUB2 or EPUB3

### ISSUE THREAD

**Issue by arthurattwell** [electricbookworks/electric-book#56]
As a technical or editorial user, I want to be able to choose between outputting EPUB2 or EPUB3, so that I can publish the most appropriate format for a particular audience.

**Comment by arthurattwell** [comment]
In addition to having differently structure `head` elements, EPUB2 and EPUB3 must treat advanced interactivity differently. Advanced interactive features must be embedded using `include` tags, so that in the `include` file we can have logic for each output format. This logic can control whether to include the advanced interactivity or substitute content (e.g. instead of a video, use an image with a link) for a given output format. Includes can support various parameters to control fallbacks, such as images for video, links to online versions, and clickable link text. Of course, even for advanced-interactivity outputs, `include`s should include fallbacks, for instance a 'Sorry' message for browsers that don't support HTML5 video. Those fallbacks can also take data from `include` tag parameters.

**Comment by arthurattwell** [comment]
I no longer believe it's worth investing time in supporting EPUB2. We should be working towards supporting finished-EPUB3 export, which means we don't need to provide an EPUB2/EPUB3 option. The current EB template will continue to output EPUB2 until we create our own EPUB3 packager to replace the need for Sigil.

Why have I changed my firmly held belief that EPUB2 is important?

* Compatibility for old devices is less important today than it was a couple of years ago, in part because of the dominant market share of EPUB3-compliant readers, and the fact that ebook vendors (like Amazon) may convert EPUB3 files to other device-compliant versions on the fly for users.
* Working around EPUB2 strictness (XHTML 1.1 especially) is hard work, and means we must avoid otherwise sensible HTML5, like `figure` elements.
* Sigil now provides a solid (enough) EPUB3 editor for our needs. For right now, converting EPUB2 to EPUB3 in Sigil is very easy.
* While I don't care for supporting Adobe Digital Editions, in those times when a client insists on it, I've found ADE's CSS support in EPUB2 is so bad it's unusable. ADE handles EPUB3 much better.

Given all that, I'm closing this issue.
